fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchReverseChainTest.java

```diff
@@ -0,0 +1,1034 @@
+    }
+
+    @Test(groups = { "server-search-reverse-chain" }, dependsOnMethods = { "testCreateLocation" })
+    public void SearchMultipleReverseChainWithLocationNear() throws Exception {
+        WebTarget target = getWebTarget();
+        Response response =
+                target.path("Endpoint")
+                .queryParam("_has:Organization:endpoint:_has:Location:organization:near", "42.256500|-83.694810|11.20|km")
+                .request(FHIRMediaType.APPLICATION_FHIR_JSON)
+                .get();
+        assertResponse(response, Response.Status.OK.getStatusCode());
+        Bundle bundle = response.readEntity(Bundle.class);
+
+        assertNotNull(bundle);
+        assertTrue(bundle.getEntry().size() == 1);
+        assertEquals(endpointId, bundle.getEntry().get(0).getResource().getId());
+    }
+
+    @Test(groups = { "server-search-reverse-chain" }, dependsOnMethods = {"testCreateProcedure1", "testCreateProcedure2"})
+    public void testSearchSingleReverseChainWithInclude() {
+        WebTarget target = getWebTarget();
+        Response response =
+                target.path("Patient")
+                .queryParam("_has:Procedure:subject:instantiates-uri", "1" + tag)
+                .queryParam("_include", "Patient:organization")
+                .request(FHIRMediaType.APPLICATION_FHIR_JSON)
+                .get();
+        assertResponse(response, Response.Status.OK.getStatusCode());
+        Bundle bundle = response.readEntity(Bundle.class);
+
+        assertNotNull(bundle);
+        assertTrue(bundle.getEntry().size() == 2);
+        List<String> resourceIds = new ArrayList<>();
+        for (Bundle.Entry entry : bundle.getEntry()) {
+            resourceIds.add(entry.getResource().getId());
+        }
+        assertTrue(resourceIds.contains(patient1Id));
+        assertTrue(resourceIds.contains(organization1Id));
+    }
+
+    @Test(groups = { "server-search-reverse-chain" }, dependsOnMethods = {"testCreateProcedure1", "testCreateProcedure2"})
+    public void testSearchSingleReverseChainWithRevInclude() {
+        WebTarget target = getWebTarget();
+        Response response =
+                target.path("Patient")
+                .queryParam("_has:Procedure:subject:instantiates-uri", "1" + tag)
+                .queryParam("_revinclude", "Procedure:patient")
+                .request(FHIRMediaType.APPLICATION_FHIR_JSON)
+                .get();
+        assertResponse(response, Response.Status.OK.getStatusCode());
+        Bundle bundle = response.readEntity(Bundle.class);
+
+        assertNotNull(bundle);
+        assertTrue(bundle.getEntry().size() == 2);
+        List<String> resourceIds = new ArrayList<>();
+        for (Bundle.Entry entry : bundle.getEntry()) {
+            resourceIds.add(entry.getResource().getId());
+        }
+        assertTrue(resourceIds.contains(patient1Id));
+        assertTrue(resourceIds.contains(procedure1Id));
+    }
+
```

Comment:
even if this fails, let's have a multiple _has test and capture the failure. 
Also probably should update conformance to state if multiple has is supported or not.

Reply:
Multiples are supported. The testcases starting with testSearchMultipleReverseChain... test this. It is stated in the FHIR spec that this is supported, but I can add a blurb to the conformance doc if you think I need to.